stm32/unittest/src/tst_api_demod.c

```diff
@@ -100,10 +102,8 @@ int main(int argc, char *argv[]) {
     struct my_callback_state my_cb_state;
     int            frame;
     int            nread, nin, nout;
-    struct MODEM_STATS stats;
     int            sync;
```

Comment:
This has grown with my recent FreeDV 2020 adventures.  Need to do something about making this usable in embedded systems.